### PR TITLE
tagging for the private link resource 

### PR DESCRIPTION
**Pull request by prahladram**
## Summary
Resolves #6 
There is no tagging for the created endpoint because of that very hard to differentiate. 

## Subscribe
- @febryantonius 
- @nuradhi 
- @rafikurnia

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -17,3 +17,24 @@ variable "service_provider_name" {
...
   description = "Service name of the PrivateLink service provider"
 }
+
+variable "service_name" {
+  type        = "string"
+  description = "Stack name of the PrivateLink"
+}
+
+variable "description" {
+  type        = "stringi"
```
Comment (febryantonius) [inline variables.tf]:
```suggestion
  type        = "string"
```

**Comment by prahladram** [comment]
@febryantonius Fixed the typo.

**Comment by mage95** [comment]
got following error when applying to my module when creating vpce

```
Error: module.vpce.aws_vpc_endpoint.service_consumer: : invalid or unknown key: tags
```

**Comment by febryantonius** [comment]
> got following error when applying to my module when creating vpce
> 
> ```
> Error: module.vpce.aws_vpc_endpoint.service_consumer: : invalid or unknown key: tags
> ```

It's most likely caused by old version of provider aws. `tags` is introduced in aws provider version `2.16.0` https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2160-june-20-2019

**Pull request merged**